src/NServiceBus.Transport.RabbitMQ/Configuration/RabbitMQTransportSettingsExtensions.cs

```diff
@@ -96,6 +96,36 @@ public static TransportExtensions<RabbitMQTransport> ConnectionString(this Trans
             return transportExtensions;
         }
 
+        /// <summary>
```

Comment:
Looks like also need a way to disable the management client usage through the legacy config API.